<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CategoryEditDialogClass</class>
 <widget class="QDialog" name="CategoryEditDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑分类</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>名称</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>父类</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="CategoryComboBox" name="comboBoxParent">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>颜色</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QtColorComboBox" name="comboBoxColors">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>用户</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="SetTextComboBox" name="comboBoxUsers">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonOK">
       <property name="icon">
        <iconset resource="Money.qrc">
         <normaloff>:/Images/apply.png</normaloff>:/Images/apply.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="icon">
        <iconset resource="Money.qrc">
         <normaloff>:/Images/editdelete.png</normaloff>:/Images/editdelete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CategoryComboBox</class>
   <extends>QComboBox</extends>
   <header>CategoryComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>QtColorComboBox</class>
   <extends>QComboBox</extends>
   <header>qtcolorcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>SetTextComboBox</class>
   <extends>QComboBox</extends>
   <header>../MyComboBox/SetTextComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Money.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonOK</sender>
   <signal>clicked()</signal>
   <receiver>CategoryEditDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>CategoryEditDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
